specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json

```diff
@@ -3111,6 +3114,15 @@
         "name": "cpk-info"
       },
       "description": "The algorithm used to produce the encryption key hash. Currently, the only accepted value is \"AES256\". Must be provided if the x-ms-encryption-key header is provided."
+    },
+    "Flush": {
```

Comment:
Curious whether this should be an extended enum instead of boolean. If there are any other possibilities in dealing with the way the file is closed then I would recommend using an enum rather than a boolean for this  parameter.